### ISSUE TITLE
Bug: contains Name returns ALL studies

### ISSUE THREAD

**Issue by will-moore** [ome/omero_search_engine#68]
Search at /searchengine//api/v1/resources/submitquery/containers/ with the following query seems to return a lot of studies (all of them)?
It should return just 1 match (It does return 1 match with `"operator": "equals").

```
{
  "query_details": {
    "and_filters": [
      {
        "name": "Name (IDR number)",
        "value": "idr0123-mota-mifish/experimentA",
        "operator": "contains",
        "resource": "project"
      }
    ],
    "or_filters": [],
    "case_sensitive": false
  }
}
```

Also: this returns a single result (expected NO results) - result is the same with "project" or "container":

```
{
   "query_details":{
      "and_filters":[
         {
            "name":"Name (IDR number)",
            "value":"idr001",
            "operator":"equals",
            "resource":"project"
         }
      ],
      "or_filters":[]
   }
}
```

**Comment by khaledk2** [comment]
I have pushed a fix for that to #57. 
I have deployed it in `idr-testing`,  please check it and let me know if you still have this issue.

**Comment by will-moore** [comment]
The issues above are fixed but I'm seeing other ones...

This only returns 1 result (`idr0018-neff-histopathology/experimentA`) when I'd expect about 10:
```
{
   "query_details":{
      "and_filters":[
         {
            "name":"Name (IDR number)",
            "value":"idr001",
            "operator":"contains",
            "resource":"container"
         }
      ],
      "or_filters":[]
   }
}
```

And this gives me `{"Error": "Your query returns no results"}`:

```
{
   "query_details":{
      "and_filters":[
         {
            "name":"Name (IDR number)",
            "value":"idr0123-mota-mifish/experimentA",
            "operator":"equals",
            "resource":"container"
         }
      ],
      "or_filters":[]
   }
}
```

**Comment by khaledk2** [comment]
I have fixed this issue and the fix has been deployed in `idr-testing`. Please test it and let me know what you think.

**Comment by will-moore** [comment]
This query (now using "name" key) returns no results:

```
{
   "query_details":{
      "and_filters":[
         {
            "name":"name",
            "value":"idr0127-baer-phenotypicheterogeneity",
            "operator":"equals",
            "resource":"container"
         }
      ],
      "or_filters":[]
   }
}
```

**Comment by will-moore** [comment]
@khaledk2 I wonder if you could add a test that uses each name from https://idr-testing.openmicroscopy.org/searchengine/api/v1/resources/all/names/ in the `submitquery/containers` search above and check that it returns the corresponding container, since it takes a while to test them all manually?

**Comment by khaledk2** [comment]
I have fixed something maybe related to this but  https://github.com/ome/omero_search_engine/issues/68#issuecomment-1337529340 should not return results because the operator is `equals`  and there are three projects which contain the provided name, e.g. `idr0127-baer-phenotypicheterogeneity/experimentA`, so `contains ` operator should be used to return the projects.
Now searchengine indexes only the images which have key-value pairs and all the images inside this screen `idr0015-colin-taraoceans/screenA`  do not have key-value pairs. I will re-index the data to include the images even if they do not have key-value pair.
I will write a to check the number of images inside each project.

**Comment by khaledk2** [comment]
You should use a `screen` or `container `instead of a `project` for the resource in the second condition.

**Comment by khaledk2** [comment]
The reason for getting results using `idr.openmicroscopy`  when setting a resource to `project` is issue #36,  it should be fixed in PR #69.

**Comment by will-moore** [comment]
Ah, I'd forgotten that had changed. I'll just use `container` for all these queries (loading images under each container). Thx

**Comment by khaledk2** [comment]
I think this has been fixed on PR #57 and #69.
